styles.css

```diff
@@ -0,0 +1,159 @@
+@import url('https://fonts.googleapis.com/css2?family=Mulish&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=DM+Sans&display=swap');
+
+body {
+    background-color: rgb(255, 255, 255);
+    font-family: 'DM Sans', sans-serif;
+}
+
+h1,
+h2,
+h3{
+    text-align: left;
+}
+
+p {
+    text-align: left;
+    font-family: 'Mulish', sans-serif;
+    font-size: 1.25em;
+}
+
+button {
+    cursor: pointer;
+}
+
+.container {
+    margin: 3% 19%;
+    text-align: center;
+}
+
+header {
+    display: flex;
+    flex-basis: 100%;
+    flex-wrap: wrap;
+    margin-bottom: 7.5em;
+}
+
+.header__logo__wrap {
+    flex-grow: 1;
+    display: flex;
+    justify-content: flex-start;
+}
+
+.header__logo {
+    height: 2.5em;
+    width: 12.5em;
+}
+
+nav {
+    flex-grow: 1;
+}
+
+nav ul {
+    display: flex;
+    justify-content: center;
+    padding: 0;
+    list-style: none;
+}
+
+nav li {
+    flex-grow: 1;
+    padding-bottom: 10px;
+    font-size: 15px;
+    font-weight: bold;
+} 
+
+nav a {
+    color: #696871;
+    text-decoration: none;
+}
+
+nav a:hover {
+    border-bottom: 0.15em solid #5454d4;
+    padding-bottom: 0.3em;
+    color: #19191b;
+}
+
+.header__button__wrap {
+    flex-grow: 1;
+    display: flex;
+    justify-content: flex-end;
+}
+
+.header__button__wrap button {
+    height: 3.8em;
+    width: 12.5em;
+    border-radius: 10px;
+    border: none;
+    background-color: #ff7143;
+    color: rgb(255, 255, 255);
+    font-size: 17px;
```

Comment:
if you start in HEX color, why you write in RGB

Reply:
Fixed